package.json

```diff
@@ -0,0 +1,23 @@
+
+{
+  "name": "react-native-android-fragment",
+  "version": "1.0.0",
```

Comment:
Just to confirm, the react-native-android-fragment version will no longer based on the react-native version it uses, correct?

Reply:
Thats what I'm thinking but open to suggestions on the approach - It could be that react native 0.47-0.60 is compatible with this library, then a breaking change would require an update. So it doesn't make sense to always create a manual update if nothing actually changes.